src/category_theory/limits/shapes/zero.lean

```diff
@@ -108,4 +109,18 @@ has_terminal_of_unique 0
 
 end has_zero_object
 
+/-- In the presence of zero morphisms, coprojections into a coproduct are (split) monomorphisms. -/
+instance
+  {β : Type v} [decidable_eq β]
+  [has_zero_morphisms.{v} C]
+  (f : β → C) [has_colimit (functor.of_function f)] (b : β) : split_mono (sigma.ι f b) :=
```

Comment:
It looks like this instance does not apply to binary coproducts (I tried `instance [has_zero_morphisms.{v} C] {X Y : C} [has_colimit (pair X Y)] : split_mono (coprod.inl : X ⟶ X ⨿ Y) := by apply_instance` and it failed). Perhaps an explicit instance for binary (co)products should be added?

Reply:
Good catch, I've added these!